test/parallel/test-promises-warning-on-unhandled-rejection.js

```diff
@@ -12,18 +12,35 @@ let b = 0;
 process.on('warning', common.mustCall((warning) => {
   switch (b++) {
     case 0:
-      assert.strictEqual(warning.name, 'UnhandledPromiseRejectionWarning');
-      assert(/Unhandled promise rejection/.test(warning.message));
+      // String rejection error displayed
+      assert.strictEqual(warning.message, 'This was rejected');
       break;
     case 1:
-      assert.strictEqual(warning.name, 'DeprecationWarning');
+      // Warning about rejection not being handled (will be next tick)
+      assert.strictEqual(warning.name, 'UnhandledPromiseRejectionWarning');
+      assert(/Unhandled promise rejection/.test(warning.message), 'Expected warning message to contain "Unhandled promise rejection" but did not. Had "' + warning.message + '" instead.');
```

Comment:
these lines need to be line-wrapped at <= 80 chars

Reply:
How come `make test`'s linter passes for me? I totally missed that.